WorksheetC

```diff
@@ -0,0 +1,25 @@
+a) The first algorithm performs the task of searching through a list and checking if there is any copies of data within that list.
+
+b) It is Quadratic because it has to search through the list to see if there is any copies of data in the list by comparing "i=0" to all other
+   possible entries within the list. It then moves on to "i=1" and does the same. It will do this for every entry in the list.
+
+c) The algorithm is still correct as it will check all the entries in the list up to the entry before "i". This means that it will check to 
+   see if there is any copies of the entry represented by "i" before that in the list it will find them. 
```

Comment:
✅ but could be clearer why this is still correct, i.e. each pair of elements is still checked at some stage